File: problem_1/python/problem1.py
#!/usr/bin/python

def threeorfive(n):
  if ( n % 3 == 0 or n % 5 == 0):
    return True
  else:
    return False

def main():
  first_list = range(1,1000)
  second_list = filter(threeorfive, first_list)
  print "%s" % (sum(second_list))

if __name__ == "__main__":
  main()

File: problem_1/perl/problem1.pl
#!/usr/bin/perl

my $count = 0;
my $total = 0;

while ( $count < 1000)
{
  if ( $count % 3 == 0 or $count % 5 == 0)
  {
    $total += $count;
  }

  $count++;
}

print $total;
